models/sql_models.yaml

```diff
@@ -1,27 +1,18 @@
-models:  
+models:
   - name: rsTracksUnionPages
     model_type: sql_template
     model_spec:
       validity_time: 24h # 1 day
       materialization:
         output_type: ephemeral
         run_type: discrete
-      single_sql: |
-        {% with Tracks = this.DeRef("inputs/rsTracks") Pages = this.DeRef("inputs/rsPages") %}
-            select user_id, anonymous_id, context_session_id, timestamp from (
-            select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,min(timestamp) as timestamp from {{Tracks}} group by user_id, anonymous_id, context_session_id, timestamp::date
-            union all 
-            select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,max(timestamp) as timestamp from {{Tracks}} group by user_id, anonymous_id, context_session_id, timestamp::date) a
-            union all
-            select user_id, anonymous_id, context_session_id, timestamp from        
-            (select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,min(timestamp) as timestamp from {{Pages}} group by user_id, anonymous_id, context_session_id, timestamp::date
-            union all 
-            select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,max(timestamp) as timestamp from {{Pages}} group by user_id, anonymous_id, context_session_id, timestamp::date) 
-        {% endwith %}
+      single_sql: "{% with Tracks = this.DeRef(\"inputs/rsTracks\") Pages = this.DeRef(\"inputs/rsPages\") %}\n    select user_id, anonymous_id, context_session_id, timestamp from (\n    select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,min(timestamp) as timestamp from {{Tracks}} group by user_id, anonymous_id, context_session_id, timestamp::date\n    union all \n    select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,max(timestamp) as timestamp from {{Tracks}} group by user_id, anonymous_id, context_session_id, timestamp::date) a\n    union all\n    select user_id, anonymous_id, context_session_id, timestamp from        \n    (select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,min(timestamp) as timestamp from {{Pages}} group by user_id, anonymous_id, context_session_id, timestamp::date\n    union all \n    select ANONYMOUS_ID,USER_ID,timestamp::date as dt,CONTEXT_SESSION_ID,max(timestamp) as timestamp from {{Pages}} group by user_id, anonymous_id, context_session_id, timestamp::date) \n{% endwith %}\n"
```

Comment:
@yajuaryarudderstack The format is a little distorted here, hope this is working as expected

Reply:
@nidhilashkari17 It is working as expected though it doesn't look good. Let me revert to earlier format.

Reply:
Ran without errors. So have pushed reverted SQL model.